### DISCUSSION TITLE
The Japanese language OSPO Local Meetup quarterly report #7 (2024/7-9)

### DISCUSSION THREAD

**Discussion by kuwatm** [todogroup/ospology#510] [OSPO Stories]
OSPO Local Meetups in Japanese are held normally twice a month in Japan.
This quarter the number of participants ranged from 5 to 13. It may look small but
we had special collaboration meetup with Cloud Native Community Japan (CNCJ) once.
We didn't have hybrid meetup in this quarter, but will have one 18th October, let's meet there!!

<< Collaboration meetup with CNCJ >> (20240801)
This meetup was held with CNCJ. And we introduce each other and one hour past quite fast.
However, a few member from CNCJ attended OSPO Local Meetup after this collaboration.

<< Strategy & Hosting >> Leader Motai-san
(20240712 No.35)(20240809 No.37)(20240913 No.39)
Motai-san has proposed to discuss about contribution from companies with each categorized industry. We have attendees from multi-industry, so asked them to fill the spread sheet out. Still there is a lot of space to discuss, but it is already valuable to get findings.
Under discussion spread sheet:
https://github.com/OpenChain-Project/OpenChain-JWG/blob/master/OSPO/OSPOLocalMeetup-Memo/QuarterlyReport/202409/OSPO%20Japan%20Local%20Meetup%20-%20OSS%20Strategy%2C%20contribution-industry.xlsx

<< OSPO Launch & FAQ >> Leader Owada-san
(20240726 No.36)(20240823 No.38)
 We discussed about education, for example "Why education should be for whole company?" and "Why education class should be yearly and mandatory?". We also discussed about contribution especially its importance and difficulty to convince engineers.
Slide currently being created:
https://github.com/OpenChain-Project/OpenChain-JWG/blob/master/OSPO/OSPOLocalMeetup-Memo/QuarterlyReport/202409/OSPO_SimpleQA_En_202409.pptx

The materials are also arranged below in Japanese and English for your perusal.
[OpenChain-JWG/OSPO/OSPOLocalMeetup-Memo at master · OpenChain-Project/OpenChain-JWG (](https://github.com/OpenChain-Project/OpenChain-JWG/tree/master/OSPO/OSPOLocalMeetup-Memo)[github.com](http://github.com/)[)](https://github.com/OpenChain-Project/OpenChain-JWG/tree/master/OSPO/OSPOLocalMeetup-Memo)

**Comment by kuwatm** [comment]
Thank you Owada-san and Muta-san for reviewing this report!

**Reply by shu-mutou** [reply]
Thank you for featuring CNCJ! :heart:
